distribution/ddf-common/src/main/resources-filtered/etc/application-definitions/admin-app.json

```diff
@@ -0,0 +1,9 @@
+{
+  "name": "admin-app",
+  "description": "Administration application for installation and management.\nIncludes the Admin UI and the underlying application service that supports the user interface.\nThe Admin UI provides contains modules allowing the administrator to access configuration pages to tailor system services and properties.::Admin",
```

Comment:
✏️can we get away from encoding the app name in the description i.e. `::Admin`

Reply:
Same comment

Reply:
Although the backend no longer uses the end of the description, the frontend is parsing it to give it a display name. I originally removed them but the app names were no longer user friendly. I was going to add a displayName field and method to the Application interface until I started deprecating it.